### DISCUSSION TITLE
Is it possible to hide the root folder?

### DISCUSSION THREAD

**Discussion by fejstan** [artgris/FileManagerBundle#104] [Q&A] [answered]
In some cases the root folder is just confusing for the user. In this case we are creating private folders based on the users id, which becomes the root folder for the user. But the user does not understand why it's a number in the root of the tree.
Is it possible to hide the root folder or maybe mask it with another label (eg. "root")?

![image](https://user-images.githubusercontent.com/3601223/173613986-23f4ab2e-6653-4ffb-a453-284784f87ae0.png)

**Comment by artgris** [comment, marked as answer]
Hi, 

release 2.5.1 add "root_name" option, to set the root folder name 

https://github.com/artgris/FileManagerBundle/blob/master/Resources/doc/book/1-basic-configuration.md#root_name-to-set-the-root-folder-name
